### DISCUSSION TITLE
How to access pgvector 0.6.0?

### DISCUSSION THREAD

**Discussion by KKosana318** [supabase/supabase#22416] [Questions] [answered]
We're using a managed Supabase database and it seems that the pgvector version is still set to 0.5. We want to run parallel index builds as mentioned in [this](https://supabase.com/blog/pgvector-fast-builds) blog post. 

Is there a way to access pgvector 0.6.0 right now? If not, is there an estimate of when we can access it?

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://github.com/supabase/supabase/assets/54564956/73ff65f1-89e7-4e79-8f08-62f0946eac30)
Just upgrade your Postgres version...

An older project with option to upgrade:
![image](https://github.com/supabase/supabase/assets/54564956/3946bc78-5cbc-4a52-a00f-d767a4fbb015)
